index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/style.css" />
+    <title>Anvito</title>
+  </head>
+  <body>
+    <header>
+      <div class="container">
+        <div class="header__wrapper">
+          <a href="index.html" class="header__logo">
+            <img src="./images/anvio-logo.png" alt="logo" />
+          </a>
+          <div class="header__nav">
+            <ul class="header__nav-list">
+              <ul class="header__nav-list upper__li">
```

Comment:
only `li` tag should be direct child of `ul`, please restructure your html a bit